### PR TITLE
[MP-414] Rewire resign step to use fastlane sigh to try to fix breakage on Bitrise with Xcode 12

### PR DESCRIPTION
**Pull request by abelyeupear**
WIP

### CODE REVIEW

**Review by elashpear — DISMISSED** [review]
⛔️ Note - please do NOT merge this PR before consulting mobile excellence group as I had to make breaking changes to the step's API that will cause build breakages for consumers, and they are all pointing straight at master AFAIK. Apps that need these changes can opt-in by pointing directly at this branch for now.

**Comment by elashpear** [comment]
should bump version as this is a breaking change

File: resigner.rb [hunk 1]
```diff
@@ -3,50 +3,38 @@
...
-		unzip_results = `/usr/bin/unzip -q "#{ipa_path}"`
-		raise "\e[31mError: #{unzip_results}\e[0m" unless $?.exitstatus.eql?(0)
-	end
-
-	def unsign
-		Dir['Payload/**/_CodeSignature'].each { |directory| FileUtils.rm_rf(directory) }
-	end
-
 	def get_distribution_type_name(distribution_type)
+        #TODO this needs to be updated to handle Apple Developer certs if we switch to those
```
Comment (edanapear) [inline resigner.rb:37]: @elashpear is this still needed?

**Review by edanapear — APPROVED** [review]

**Comment by edanapear** [comment]
Do NOT delete the branch

**Pull request merged**